docs/styleguide.md

```diff
@@ -250,10 +250,80 @@ The properties chapter must be in following form:
 The heading can be `###` or `####`-levels depending on whether the property
 belongs to a module or a class.
 
+## API History
+
+An "API History" block is a YAML code block encapsulated by a HTML comment that
+should be placed directly after the Markdown header for a class or method, like so:
+
+`````markdown
+#### `win.setTrafficLightPosition(position)` _macOS_
+
+<!--
+```YAML history
+added:
+  - pr-url: https://github.com/electron/electron/pull/22533
+changes:
+  - pr-url: https://github.com/electron/electron/pull/26789
+    description: Made `trafficLightPosition` option work for `customButtonOnHover` window.
+deprecated:
+  - pr-url: https://github.com/electron/electron/pull/37094
+    breaking-changes-header: deprecated-browserwindowsettrafficlightpositionposition
+```
+-->
+
+* `position` [Point](structures/point.md)
+
+Set a custom position for the traffic light buttons. Can only be used with `titleBarStyle` set to `hidden`.
+`````
+
+It should adhere to the API History [JSON Schema](https://json-schema.org/)
+(`api-history.schema.json`) which you can find in the `docs` folder.
+The [API History Schema RFC][api-history-schema-rfc] includes example usage and detailed
+explanations for each aspect of the schema.
+
+The purpose of the API History block is to describe when/where/how/why an API was:
+
+* Added
+* Changed (usually breaking changes)
+* Deprecated
+
+Each API change listed in the block should include a link to the
+PR where that change was made along with an optional short description of the
+change. If applicable, include the [heading id](https://gist.github.com/asabaylus/3071099)
+for that change from the [breaking changes documentation](./breaking-changes.md).
+
+The [API History linting script][api-history-linting-script] (`lint:api-history`)
+validates API History blocks in the Electron documentation against the schema and
+performs some other checks. You can look at its [tests][api-history-tests] for more
+details.
+
+There are a few style guidelines that aren't covered by the linting script:
+
+* Always adhere to this format:
+
+  | Type                        | Line                              |
+  |----------------------------:|:----------------------------------|
+  | `API HEADER`                | `` #### `win.flashFrame(flag)` `` |
+  | `BLANK LINE`                | `​`                                |
+  | `HTML COMMENT OPENING TAG`  | `` <!-- ``                        |
+  | `API HISTORY OPENING TAG`   | `` ```YAML history ``             |
+  | `API HISTORY`               | `added: ...`                      |
+  | `API HISTORY CLOSING TAG`   | `` ``` ``                         |
+  | `HTML COMMENT CLOSING TAG`  | `` --> ``                         |
+  | `BLANK LINE`                | `​`                                |
+
+* Use two spaces for indentation in the YAML.
+* Keep change descriptions concise.
```

Comment:
```suggestion
* Keep change descriptions concise - favor using the release notes from the associated PR whenever possible.
```

I think most of the time we want the description to be the release notes (possibly with minor grammatical changes)